festim/materials/materials.py

```diff
@@ -7,15 +7,15 @@
 from typing import Union
 
 
-class Materials:
+class Materials(list):
     """
     Args:
         materials (list, optional): contains festim.Material objects.
```

Comment:
Documentation needs updating cause it doesn't have a `materials` argument anymore.
I'd simply have

```python
a list of festim.Material objects
```

Reply:
Right